docs/topology.md

```diff
@@ -33,13 +33,9 @@ including offering serialization to a variety of standard formats (including [XM
     Particle
     Atom
     Bond
-    VirtualSite
-    VirtualParticle
-    BondChargeVirtualSite
-    MonovalentLonePairVirtualSite
-    DivalentLonePairVirtualSite
-    TrivalentLonePairVirtualSite
     ValenceDict
     ImproperDict
+    ~molecule.HierarchyScheme
+    ~molecule.HierarchyElement
```

Comment:
Let's re-export this at the `topology` level so that we don't have to do the `~molecule` thing here